src/src/PluginStructs/P104_data_struct.cpp

```diff
@@ -1801,27 +1811,27 @@ bool P104_data_struct::webform_load(struct EventStruct *event) {
     fontCount++;
     # endif // ifdef P104_USE_KATAKANA_FONT
     const __FlashStringHelper *fontTypes[] = {
-      F("Default")
+      F("Default (0)")
     # ifdef P104_USE_NUMERIC_DOUBLEHEIGHT_FONT
-      , F("Numeric, double height")
+      , F("Numeric, double height (1)")
     # endif   // ifdef P104_USE_NUMERIC_DOUBLEHEIGHT_FONT
     # ifdef P104_USE_FULL_DOUBLEHEIGHT_FONT
-      , F("Full, double height")
+      , F("Full, double height (2)")
     # endif   // ifdef P104_USE_FULL_DOUBLEHEIGHT_FONT
     # ifdef P104_USE_VERTICAL_FONT
-      , F("Vertical")
+      , F("Vertical (3)")
     # endif   // ifdef P104_USE_VERTICAL_FONT
     # ifdef P104_USE_EXT_ASCII_FONT
-      , F("Extended ASCII")
+      , F("Extended ASCII (4)")
       # endif // ifdef P104_USE_EXT_ASCII_FONT
     # ifdef P104_USE_ARABIC_FONT
-      , F("Arabic")
+      , F("Arabic (5)")
     # endif   // ifdef P104_USE_ARABIC_FONT
     # ifdef P104_USE_GREEK_FONT
-      , F("Greek")
+      , F("Greek (6)")
     # endif   // ifdef P104_USE_GREEK_FONT
     # ifdef P104_USE_KATAKANA_FONT
-      , F("Katakana")
+      , F("Katakana (7)")
```

Comment:
Maybe the font specific checks/defines etc. can be moved to a separate class (or 2nd cpp file) ?
Now the various #ifdef checks are spread all of the rather long file.
That makes it hard to check if all are handled.

Reply:
I did have a look at taking that code to a separate class or source file, but it currently exceeds my c++ knowledge to achieve that.
For now I'll further enhance the documentation on adding a font definition to exactly describe what has to be done and where.

Reply:
Check!